### DISCUSSION TITLE
Fail no nuxt 3 deploy

### DISCUSSION THREAD

**Discussion by mitraecp** [vercel/community#1790] [Help] [answered]
### Summary

I guys, i have a problem on my nuxt 3 deploy, it was working ok, but now i got this error

### Additional information

![image](https://user-images.githubusercontent.com/75841640/225299616-cef0460d-2071-4ade-abfa-c87cad31e4f7.png)

**Comment by andreaminieri** [comment, marked as answer]
I had the same error with a Nuxt 2 project on Vercel. This could be related to nuxt/nuxt#19717. This pull request nuxt-modules/web-vitals#64 fixed it for me.
